extensions/filehost.md

```diff
@@ -27,11 +27,11 @@ This specification offers a way for network administrators to recommend a hostin
 
 ## Architecture
 
-This specification introduces the `draft/FILEHOST` isupport token.
+This specification introduces the `draft/FILEHOST` RPL_ISUPPORT token. Please see https://modern.ircdocs.horse/#feature-advertisement for what an RPL_ISUPPORT token is.
```

Comment:
I don't think that is needed. Other specs don't link to it either, and googling "irc rpl_isupport" gives you good results.

Reply:
Yeah, true.